docs/source/overview.rst

```diff
@@ -13,12 +13,13 @@ There are two procedures:
 Features
 --------
 
-* No need to specify enquiries period (there is no *active.enquiries* status), since it overlaps with *active.tendering* period.
+* Organizer can provide only the *Tender.auctionPeriod.startDate* as the base date for the rest periods (they will be calculated automatically).
```

Comment:
I would rather suggest to use the statement provided in line 21

Reply:
The original one?
This
```
The only date Organizer has to provide is *Tender.auctionPeriod.startDate*, the rest will be calculated automatically.
```
?